src/Common/StackTrace.cpp

```diff
@@ -230,26 +238,41 @@ void StackTrace::symbolize(
                 auto dwarf_it = dwarfs.try_emplace(object->name, object->elf).first;
 
                 DB::Dwarf::LocationInfo location;
-                std::vector<DB::Dwarf::SymbolizedFrame> inline_frames;
                 if (dwarf_it->second.findAddress(
-                        uintptr_t(current_frame.physical_addr), location, DB::Dwarf::LocationInfoMode::FAST, inline_frames))
+                        uintptr_t(current_frame.physical_addr), location, mode, inline_frames))
                 {
                     current_frame.file = location.file.toString();
                     current_frame.line = location.line;
                 }
             }
         }
-        else
-            current_frame.object = "?";
 
         if (const auto * symbol = symbol_index.findSymbol(current_frame.virtual_addr))
             current_frame.symbol = demangle(symbol->name);
-        else
-            current_frame.symbol = "?";
```

Comment:
what will happen now with frames that were displayed as `'?'`?

Reply:
I've removed this from the forEacFrame because it is generic code, but this code still exists in `toStringEveryLineImpl`:

```
          if (frame.symbol.has_value() && frame.file.has_value())                                                                                                                                                    
              out << demangleAndCollapseNames(*frame.file, frame.symbol->data());                                                                                                                                    
          else                                                                                                                                                                                                       
              out << "?";                                                                                                                                                                                            
```